metadata.json

```diff
@@ -1,34 +1,56 @@
 {
-  "name": "puppetlabs-puppet_authorization",
-  "version": "0.5.0",
-  "author": "puppetlabs",
+  "name": "puppet-puppet_authorization",
+  "version": "0.6.0",
```

Comment:
please don't update the version. we do this during the release process

Reply:
Reverted!